concordium-node/src/stats_export_service.rs

```diff
@@ -299,6 +326,17 @@ impl StatsExportService {
         )?;
         registry.register(Box::new(grpc_request_response_time.clone()))?;
 
+        let grpc_pending_requests_gauge = IntGauge::with_opts(Opts::new(
```

Comment:
I don't think "pending" is an accurate description. It is requests in flight as the tower-http metric is named.

Pending to me implies that it is waiting to be  replied, but that is not what we are tracking here.

Reply:
I have renamed it now